spec/unit/pops/serialization/to_from_hr_spec.rb

```diff
@@ -296,10 +296,15 @@ def parse(string)
           expect(val2).to be_a(Types::PObjectTypeExtension)
           expect(val2).to eql(val)
         end
+
+        it 'with POjbectTypeExtension type being converted' do
+          val = { 'ObjectExtension' => type.create(34) }
+          expect(to_converter.convert(val))
+            .to eq({"ObjectExtension"=>{"__ptype"=>"MyType", "x"=>34}})
```

Comment:
nit, the outer `{...}` are redundant, so it could just be `eq("ObjectExtension"=>{..})`. But I see the spec file uses them everywhere, so feel free to ignore.